.github/workflows/pre-release.yml

```diff
@@ -10,30 +10,49 @@ jobs:
     name: "Pre-release"
     runs-on: "ubuntu-latest"
     container: ghcr.io/opencs-ontology/ci-worker:main
-
+    
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
         with:
           path: opencs
 
       - name: "Package ontology"
-        run: python /app/package.py opencs package dev
-
+        run: python /app/package.py opencs package dev;
+          
+      - name: "Download robot.jar"
+        uses: wei/wget@v1
+        with:
+          args: https://github.com/ontodev/robot/releases/download/v1.9.1/robot.jar
+        
+      - name: "Java installation"
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin' 
+          java-version: '17'   
+          
```

Comment:
Lines 21–33 will be executed on every CI run. So, every time someone changes something in OpenCS, the CI has to download some (specific) version of ROBOT and some version of Java. This is not very efficient.

The steps needed to install Java and ROBOT should be moved to the ci-worker's Dockerfile, in my opinion. This will have two advantages: 1. we don't download this stuff every time, speeding up the workflow dramatically; 2. we can re-use ROBOT in other workflows easily, which will surely come in handy (it's a very good and robust tool). So, I'd recommend you start by modifying the ci-worker accordingly.

Reply:
Line 21 is actually not the line from our part of pipeline. Our part starts at line 23. All our steps before inference take 24 seconds (wget, download robot.jar + java ) so we think it is not that long. 
![image](https://user-images.githubusercontent.com/56300703/213865498-6610d890-2a73-4f12-8eed-630517cfe1b5.png)
![image](https://user-images.githubusercontent.com/56300703/213865505-ece949a2-d5e4-428e-9a5d-9fa79cd4ee96.png)

We are aware that time of execution of the pipeline could be shortened using your way of handling it, but we are not experienced enough to do this by ourselves. As it would take us long period of time to learn how to do it your way and fix problem, we could do it apart from project. And for now we would ask you to accept this solution.

Reply:
OK, you did the homework to convince me. :)

Please open an issue in this repo that is about moving these downloads to ci-worker's Dockerfile.  Then I will accept your solution.